.github/workflows/temp.yml

```
@@ -0,0 +1,39 @@
+          retry-count: 3
+          retry-delay: 500
+
+      - name: Print outputs
+        run: |
+          echo "Status: ${{ steps.request.outputs.status }}"
+          echo "Success: ${{ steps.request.outputs.success }}"
+          echo "Headers: ${{ steps.request.outputs.headers }}"
+          echo "Body: ${{ steps.request.outputs.body }}"
+          echo "Artifact downloaded successfully as artifact.zip"
+          # Optional: Save the body (artifact) to a file if it is binary content
+          echo "${{ steps.request.outputs.body }}" > artifact.zip
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)